### DISCUSSION TITLE
What kinda PR are meaningful for this project ?

### DISCUSSION THREAD

**Discussion by uzair004** [vladmandic/human#144] [General]
I am playing around with this project for a while to implement in my project.

I have taken node-demo and edit it according to my need.

I believe i can add two things to node demo file

1. Adding buffer support (will accept buffer as well)
2. Reorganize the code, each operation will have it's own function. Main() and detect() will look bit cleaner

**Comment by vladmandic** [comment]
adding buffer support is welcome.
complete reorg? if done nicely, why not. but at the end, its just a demo - the magic is in the library itself, not the demo.

but as a rule i do welcome PRs!
